.github/workflows/e2e.yaml

```diff
@@ -37,11 +37,6 @@ jobs:
       run: npm ci
       working-directory: ./client
 
-    - name: Set up JDK 11
```

Comment:
This section is remove here and in the `server-gradle.yml` because the appropriate JDK is now downloaded by Gradle if necessary.

Reply:
Any idea where it puts the downloaded JDK so we can cache it?

Reply:
Nevermind, I looked it up in the docs. See https://github.com/UMM-CSci-3601/3601-iteration-template/pull/256#issuecomment-757048506